qiskit/circuit/tools/pi_check.py

```diff
@@ -139,6 +139,9 @@ def normalize(single_inpt):
 
     if real == '0' and imag != '0':
         return imag + 'j'
-    elif real != 0 and imag != '0':
-        return '{}+{}j'.format(real, imag)
+    elif real != '0' and imag != '0':
+        if float(imag) < 0:
```

Comment:
This doesn't actually work. Try
```
qc = QuantumCircuit(1)
qc.u1(complex(-1.0, -2*pi/3), 0)
qc.draw('mpl')
```
I think it's sufficient to check that the first char in imag is '-'.

Reply:
You're right. I made it more complicated

Reply:
Feel free to proceed with this if you'd like to. There's a bug in what the other person did, and they shouldn't have submitted a PR when there was already one being worked on.